tests/backends/base/test_operations.py

```diff
@@ -1,8 +1,60 @@
 from django.db import NotSupportedError, connection
+from django.db.backends.base.operations import BaseDatabaseOperations
+from django.db.models import DurationField
 from django.test import SimpleTestCase, skipIfDBFeature
 
 
 class DatabaseOperationTests(SimpleTestCase):
+    def setUp(self):
+        self.ops = BaseDatabaseOperations(connection=connection)
```

Comment:
Is there a difference between using `BaseDatabaseOperations(connection=connection)` and `connection.ops`? I guess we should use one style consistently.

Reply:
In this case, yes. `connection.ops` is the instantiated `ops_class` from `DatabaseWrapper` and won't be this base class for an actual connection. `@override_settings(DATABASES=None)` sets it to the dummy backend, but `quote_name` has a different value for the operations there.